dashboard/test/ui/step_definitions/steps.rb

```diff
@@ -325,3 +329,12 @@ def log_in_as(user)
   # Note: Safari webdriver does not support actions API
   @browser.action.key_down(:control).send_keys(key).key_up(:control).perform
 end
+
+And(/^I press keys "([^"]*)" for element "([^"]*)"$/) do |key, selector|
+  if key === ":backspace"
+    key = :backspace
+  end
+  element = @browser.find_element(:css, selector)
+  # Note: Safari webdriver does not support actions API
```

Comment:
Since this doesn't use `@browser.action` like the method above we should be able to remove this comment here